packages/web-runtime/src/pages/account.vue

```diff
@@ -241,24 +266,143 @@ export default defineComponent({
       return configurationManager.logoutUrl
     })
 
-    onMounted(() => {
-      if (unref(isLanguageSupported)) {
-        loadAccountBundleTask.perform()
+    const updateSelectedLanguage = async (option) => {
+      const settingsId = unref(valuesList).find((cV) => cV.identifier.setting === 'language')?.value
+        ?.id
+
+      const value = {
+        bundleId: unref(bundlesList)?.id,
+        settingId: unref(bundlesList)?.settings.find((s) => s.name === 'language')?.id,
+        resource: { type: 'TYPE_USER' },
+        listValue: { values: [{ stringValue: option.value }] },
+        ...(settingsId && { id: settingsId })
+      }
+
+      try {
+        await axios.post(
+          '/api/v0/settings/values-save',
+          { value: { ...value, accountUuid: 'me' } },
+          {
+            headers: {
+              authorization: `Bearer ${unref(accessToken)}`,
+              'X-Request-ID': uuidV4()
+            }
+          }
+        )
+
+        selectedLanguageValue.value = option
+        setCurrentLanguage({
+          language,
+          languageSetting: {
+            identifier: {
+              extension: 'ocis-accounts',
+              bundle: 'profile',
+              setting: 'language'
+            },
+            value
+          }
+        })
+
+        /**
+         * Edge case: we need to reload the values list to retrieve the settingsId if not set,
+         * otherwise the backend saves multiple entries
+         */
+        if (!settingsId) {
+          loadValuesList.perform()
+        }
+      } catch (e) {
+        console.error(e)
+        store.dispatch('showMessage', {
+          title: $gettext('Unable to save language…'),
+          status: 'danger'
+        })
+      }
+    }
+
+    const updateDisableEmailNotifications = async (option) => {
+      option = !option
+      console.log(option)
+      const bundle = loadBundlesList.last?.value
+
+      const settingsId = unref(valuesList).find(
+        (cV) => cV.identifier.setting === "'disable email notifications"
```

Comment:
Is the `'` there on purpose?
If yes, can you add a comment why it's there? It *looks* wrong

Reply:
no + not ready to review =)